### PR TITLE
Service offline page

### PR DESCRIPTION
**Pull request by tomlovesgithub**
## What's included?
Service offline page improved. 
GDS/JAC Styling used.
Advised wording added.

## Who should test?
✅ Product owner
✅ Developers

## How to test?

- Switch to branch locally 
- Open apply/service-offline/index.html with any browser

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

image

--- 

PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by HalcyonJAC — APPROVED** [review]
Code looks great!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "application-form",
-  "version": "1.54.0",
+  "version": "1.53.2",
```
Comment (warrensearle) [inline package.json:3]: Please check 😄

File: service-offline/index.html [hunk 2]
```diff
@@ -1,12 +1,64 @@
...
+        <main class="govuk-main-wrapper">
+          <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+              <h1 class="govuk-heading-l">The Judicial Appointments Commission vacancy platform is currently inaccessible.</h1>
+              <h2 class="govuk-heading-m">
+                We are working to get the platform back online.
+              </h2>
+              <p class="govuk-body">
+                If you have queries about any specific roles, including about your application, please contact enquiries@judicialappointments.gov.uk.
+                <br>
```
Comment (warrensearle) [inline service-offline/index.html:49]: Indentation is a little off

**Review by warrensearle — CHANGES_REQUESTED** [review]
Looks good, just a couple of niggles to tidy

**Comment by andrew-isaac** [comment]
@tomlovesgithub  - Changes from comms team

**Comment by andrew-isaac** [comment]
@tomlovesgithub  - Can we have a new preview URL please? We would like to review this and push it out

**Review by warrensearle — APPROVED** [review]

**Pull request merged**